pkg/discord/main.go

```diff
@@ -771,6 +795,145 @@ func fetchFile(data shared.CommandData) (*models.File, error) {
 	}
 }
 
+// Check common time formats
+func parseTime(timeStr string) (time.Time, error) {
+	now := time.Now()
+	formats := []string{
+		// Time only
+		"3:04PM",
+		"3:04:05PM",
+		"15:04",
+		"15:04:05",
+		// Date & time
+		"Jan 2 3:04PM",
+		"Jan 2 3:04:05PM",
+		"Jan 2 15:04",
+		"Jan 2 15:04:05",
+		"1/2 3:04PM",
+		"1/2 3:04:05PM",
+		"1/2 15:04",
+		"1/2 15:04:05",
+	}
+	var outTime time.Time
+	var err error
+	for i, format := range formats {
+		if t, err := time.Parse(format, timeStr); err == nil {
+			if i <= 3 {
+				outTime = time.Date(now.Year(), now.Month(), now.Day(), t.Hour(), t.Minute(), t.Second(), t.Nanosecond(), now.Location())
+			} else {
+				outTime = time.Date(now.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second(), t.Nanosecond(), now.Location())
```

Comment:
now.Location() will be an issue if different time zones

Reply:
DiscordAPI may have the ability to check what time zone a User is in when a message is sent. If time zone is something you want to worry about. You could possibly check it there.

Reply:
It is. I should probably change both start and end to use the timezone specified by the brigade to simplify.

Reply:
Oh for sure. I didn't realize a brigade had an associated time zone.